shortcircuit-api/src/main/java/com/powsybl/shortcircuit/FaultResult.java

```diff
@@ -41,4 +49,10 @@ public float getThreePhaseFaultCurrent() {
         return threePhaseFaultCurrent;
     }
 
+    /**
+     * List of contributions of each connectable connected to the equipment to the three phase fault current
+     */
+    public List<ContributionResult> getContributionResults() {
+        return contributionResults;
+    }
```

Comment:
Do you think it could be useful to have a method that gives the contribution given the id of the feeder ?

Reply:
Yes! It's a good idea, I will add it